ckan/lib/i18n.py

```diff
@@ -381,10 +381,20 @@ def build_js_translations():
 
     # Build translations for each language
     for lang in sorted(langs):
-        po_files = [os.path.join(i18n_dir, lang, u'LC_MESSAGES',
-                                 domain + u'.po')
-                    for i18n_dir, domain in i18n_dirs.iteritems()]
-        po_files = [fn for fn in po_files if os.path.isfile(fn)]
+        po_files = [
+            fn for fn in (
+                os.path.join(
+                    i18n_dir,
+                    lang,
+                    u'LC_MESSAGES',
+                    domain + u'.po'
+                )
+                for i18n_dir, domain in i18n_dirs.iteritems()
+            ) if os.path.isfile(fn)
+        ]
```

Comment:
@torfsen I'm with you, but @TkTech has some pretty strongly-held beliefs around coding style :grinning: